### DISCUSSION TITLE
transfer learning LoFTR; what do I use for my data indices?

### DISCUSSION THREAD

**Discussion by amtchell** [zju3dv/LoFTR#294] [Q&A]
Hi,

I am interested in using LoFTR on my custom image dataset for additional training (using the published weights as a starting point). It is unclear to me looking at the .npz files for both scannet and meganet what data I need to include about my dataset in the indices files so that I may perform transfer learning.

Has anyone successfully used transfer learning with LoFTR? 

Thanks!

**Comment by 6152258** [comment]
您好，我现在也遇到了这样的问题，请问您成功地将迁移学习与 LoFTR 结合使用了嘛？
